src/js/components/loaders/dnaLoader.jsx

```diff
@@ -0,0 +1,41 @@
+"use strict";
```

Comment:
Is this component meant for importing a gene entry? Not really sure what this is suppose to do atm but it looks like a placeholder.